programs/CMakeLists.txt

```diff
@@ -19,20 +19,18 @@ foreach (program ${PROGRAMS})
 			${CMAKE_SOURCE_DIR}/programs/common.c 
 			${CMAKE_SOURCE_DIR}/programs/${program}.c
 			)
-	target_link_libraries(${program} sndfile)
+	target_link_libraries(${program} PRIVATE sndfile)
```

Comment:
What is the reason we link these privately?  Is there a benefit over a different type of linking?